12/umbraco-cms/reference/configuration/webroutingsettings.md

```diff
@@ -100,3 +100,5 @@ Will set the URL provider mode, options are:
 ## Umbraco application URL
 
 Defines the Umbraco application URL that the server should reach itself. By default, Umbraco will guess that URL from the first request made to the server. Use this setting if the guess is not correct (because you are behind a load-balancer, for example). Format is: `http://www.mysite.com/`, ensure to contain the scheme (http/https) and complete hostname.
+
+Note that previously it was also required to specify BackOffice path as this was customizable (`/umbraco` by default), but this is no longer possible in Umbraco 9+ so using the full URL including scheme and hostname is sufficient.
```

Comment:
We have a styleguide in place to make the documentation more readable, and one is to write sentences that are[ less than 25 words](https://docs.umbraco.com/welcome/contribute/style-guide#long-sentences). Therefore I am suggesting the below, with the addition of an info warning similar to [this](https://docs.umbraco.com/umbraco-cms/reference/configuration/webroutingsettings#:~:text=Are%20you%20on%20Umbraco%20Cloud%3F). I have suggested a new sentence on 2nd part to match the same wording as [here](https://our.umbraco.com/Documentation/Reference/Configuration-for-Umbraco-7-and-8/umbracoSettings/#:~:text=the%20scheme%20(http/https)%2C%20complete%20hostname)
```suggestion
{% hint style="info" %}
Previously before v9, it was required to specify **backofffice** path as this was customizable (`/umbraco` by default). However, from v9+ this is no longer possible, so it's sufficient to use the URL that contains the scheme (http/https) and complete hostname.
{% endhint %}
```

Reply:
I have updated this :)